algorithms/class1.html

```diff
@@ -0,0 +1,975 @@
+            </section>
+
+            <section>
+                <h1>Algorithms</h1>
+                <p><em>"A repeatable process for determining the solution to a problem."</em></p>
+            </section>
+
+            <section>
+                <h3>Algorithms in Daily Life</h3>
+                <p >Some problems that we can create algorithms for (not necessarily related to computers)</p>
+                <br />
+                <ul>
+                    <li class="fragment">Finding a fruit in the grocery store.</li>
+                    <li class="fragment">Alphabetizing name tags.</li>
+                    <li class="fragment">Finding keys that you lost.</li>
+                    <li class="fragment">Finding something good to watch on TV.</li>
+                    <li class="fragment">Deciding on a place to eat lunch.</li>
+                </ul>
+            </section>
+
+
+            <section>
+                <h3>Finding something to watch on TV v1</h3>
+                <ol class="left">
+                     <li>Turn on TV</li>
+                     <li>Watch what is on</li>
+                </ol>
+                <br><br>
+                <img src="images/tv_1_code.png" style="width:50%;">
+            </section>
+
+            <section>
+                <h3>Finding something to watch on TV v2</h3>
+                <ol class="left">
+                     <li>Turn on TV</li>
+                     <li>Flip through every channel and rate what is on</li>
+                     <li>Find the highest rated channel</li>
+                     <li>Watch</li>
+                </ol>
+            </section>
+
+            <section>
+                <img src="images/tv_2_code.png" style="width:80%;">
+            </section>
+
+             <section>
+                <h3>Finding something to watch on TV v3</h3>
+                <ol class="left">
+                     <li>Turn on TV</li>
+                     <li>Check 5 favorite channels and rate what is on</li>
+                     <li>Find the highest rated channel</li>
+                     <li>Watch</li>
+                </ol>
+            </section>
+
+            <section>
+                <img src="images/tv_3_code.png" style="width:80%;">
+            </section>
+
+             <section>
+                <h3>Which version is best?</h3>
+                <br>
+                <span class="">
+                     <p class="">Turn on the TV</p>
+                     <p class="">Rate all channels</p>
+                     OR
+                     <p class="">Rate top channels</p>
+                </span>
+            </section>
+
+            <!-- START  OF GOAL OF ALGORITHMS -->
+
+            <section>
+              <h2>Goal of Algorithms</h2>
+              <br>
+              <ul>
+                <li class="fragment">Solve a problem</li>
+                <li class="fragment">in a repeatable way</li>
+            </ul>
+            </section>
+
+            <!-- TODO move elsewhere -->
```

Comment:
This might be good to add in the last class as an overview or recap and how they're used?